pkg/config/process.go

```diff
@@ -69,4 +93,25 @@ func setupProcesses(config Config) {
 		"DD_PROCESS_AGENT_DISCOVERY_ENABLED",
 	)
 	procBindEnvAndSetDefault(config, "process_config.process_discovery.interval", 4*time.Hour)
+
+	AddOverrideFunc(loadProcessTransforms)
+}
+
+// loadProcessTransforms loads transforms associated with process config settings. This is used to handle deprecated settings
+func loadProcessTransforms(config Config) {
+	if config.IsSet("process_config.enabled") {
+		log.Debug("process_config.enabled is deprecated, use process_config.container_collection.enabled" +
+			" and process_config.process_collection.enabled instead")
+		procConfigEnabled := strings.ToLower(config.GetString("process_config.enabled"))
```

Comment:
Since we have the overwrite on `process_config.enabled` do we need the `ToLower` ?

Reply:
This is the function for applying the override; at this point we have not finished applying it, so we still need to lowercase the user's input, in case they type something like "Disabled" or "FaLse"